### DISCUSSION TITLE
Provide a way to create custom admonitions

### DISCUSSION THREAD

**Discussion by dbitouze** [executablebooks/MyST-Parser#802] [Ideas]
It would be nice for `MyST-Parser` to provide a way to create custom admonitions. 

My use case is the following: at the bottom of all the numerous (more than 1220) `.md` source files of my (LaTeX FAQ) project, I want to specify the sources of the answers to the question. I want these sources to be displayed in an  admonition, say:

- which is collapsible,
- with “Source” as title,
- with the `history` octicon (from `sphinx-design`).

It would be great to be able to create my own `source` admonition in order to be able to type, instead of a systematic:
```markdown
:::{dropdown} Source
:icon: history
⟨the source⟩
:::
```
just:
```markdown
:::{source}
⟨the source⟩
:::
```
which would be as easy as:
```markdown
:::{tip}
Let's give readers a helpful hint!
:::
```

**Comment by dbitouze** [comment]
OK, I'll manage by looking at the `MyST` code: thanks!

**Reply by chrisjsewell** [reply]
heya, I can see the use , but I feel this would be more of a question for sphinx in general, rather than specific to myst-parser
